PowerEditor/src/MISC/md5/md5Dlgs.cpp

```diff
@@ -22,13 +22,16 @@
 #include "CustomFileDialog.h"
 #include "Parameters.h"
 #include <shlwapi.h>
+#include "resource.h"
 
 INT_PTR CALLBACK HashFromFilesDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM /*lParam*/)
```

Comment:
`LPARAM /*lParam*/)` should be uncommented.

Reply:
fixed